test/module/irohad/torii/processor/mst_notifier_test.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Where are includes and `ConsensusStatusProcessorTest` fixture? How it works?

Reply:
It seems that it doesn't work. I am going to fix it.